daffodil-test/src/test/resources/org/apache/daffodil/section07/variables/variables.tdml

```diff
@@ -2217,6 +2217,9 @@
         </root>
       </tdml:dfdlInfoset>
     </tdml:infoset>
+    <tdml:warnings>
+      <tdml:warning>State was: VariableRead</tdml:warning>
```

Comment:
This should be moved up into the other `<tdml:warnings>` tag, makes it easier if all the warnings are in the same place, and there really isn't anything gained by having them separate--I didn't even realize we supported that.